src/ucp/rndv/rndv.c

```diff
@@ -1500,10 +1489,10 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_ats_handler,
     return UCS_OK;
 }
 
-ucs_status_t ucp_rndv_rts_handle_status_from_pending(ucp_request_t *sreq,
-                                                     ucs_status_t status)
+ucs_status_t ucp_rndv_send_handle_status_from_pending(ucp_request_t *sreq,
+                                                      ucs_status_t status)
 {
-    /* we rely on the fact that the RTS isn't being sent by an AM Bcopy multi */
+    /* We rely on the fact that the RTS isn't being sent by an AM Bcopy multi */
```

Comment:
minor: 
maybe mention RTR in the commant as well? Like
 `RTS and RTR should not be sent by AM bcopy multi`

Reply:
done